picotlsvs/picotls/wincompat.h

```diff
@@ -2,7 +2,8 @@
 #define WINCOMPAT_H
 
 #include <stdint.h>
-#define ssize_t int
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
```

Comment:
What errors do we have with `ssize_t -> int`?

Reply:
including this header and af_msvc.h which comes from the hydra includes creates a problem since it will have:

`#define ssize_t int` from here and `typedef SSIZE_T ssize_t;` from af_msvc.h which will end up as `typedef SSIZE_T int`

also it seems that `typedef SSIZE_T ssize_t;` is the correct solution anyways